packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -24,33 +37,144 @@ export class MetadataOutlineProvider
     this.defaultOrg = defaultOrg;
   }
 
-  public async refresh(defaultOrg?: string): Promise<void> {
-    if (!isNullOrUndefined(defaultOrg)) {
-      this.defaultOrg = defaultOrg;
+  public async onViewChange() {
+    if (await this.getDefaultUsernameOrAlias()) {
+      this.internalOnDidChangeTreeData.fire();
     }
+  }
+
+  public async refresh(defaultOrg?: string): Promise<void> {
```

Comment:
Can we visually indicate to the user that a refresh of the org browser data is happening ? Right now, if a user clicks the refresh button there's nothing that lets the user know if the feature is actually refreshing or when it's done refreshing the data.

Reply:
Hmm yeah that makes sense. I'll look into how to indicate the progress.

Reply:
Whether there's an org switch or refresh, you could clear the tree and present a non-collapsable "loading" node.